src/SignalR/server/StackExchangeRedis/test/TestConnectionMultiplexer.cs

```diff
@@ -307,7 +320,7 @@ public async Task<long> PublishAsync(RedisChannel channel, RedisValue message, C
 
         public void Subscribe(RedisChannel channel, Action<RedisChannel, RedisValue> handler, CommandFlags flags = CommandFlags.None)
         {
-            _server.Subscribe(channel, handler, flags);
+            throw new NotImplementedException();
```

Comment:
Couldn't we just associate the id with then handler somehow and avoid the private reflection? If the `Action<RedisChannel, RedisValue>` instance is not unique to an `ISubscriber` instance to begin with, maybe we could just always wrap it into a newly-allocated `Action<RedisChannel, RedisValue>` in this method. Then we could have a dictionary for handler-to-subscriber lookup.

Reply:
This method isn't used by SignalR. We use the Subscribe that returns a `ChannelMessageQueue`.